Assets/KoboldKare/Scripts/SceneDescriptor.cs

```diff
@@ -24,7 +24,17 @@ public class SceneDescriptor : OrbitCameraPivotBase {
     private OrbitCamera orbitCamera;
 
     private void Awake() {
-        instance = this;
+        //Check if instance already exists
+        if (instance == null) {
```

Comment:
I'm worried about this change, because scene descriptors, while being a singleton, need to allow the new scene to describe itself. I think it may still work since they don't persevere through scene loads though. Can you verify that this works over multiple scene loads of different maps?

Reply:
It seems to work fine for me!